src/scripts.js

```diff
@@ -45,21 +52,24 @@ const pageLoad = () => {
   showSleepInfo();
 };
 
+const printList = (parent, array) => {
+  array.forEach(item => {
+    let li = document.createElement('li');
+    parent.appendChild(li);
+    li.innerHTML += item;
+  }
+)}
+
 const showUserInfo = () => {
   const userFriends = currentUser.friends.map(friend => {
     const userFriend = userRepo.findUser(friend)
     return userFriend.name
   })
   userInfoDisplay.innerHTML = `<li class="user-info__stride-length">Stride: ${currentUser.strideLength}</li>
-  <li class="user-info__friends"> Friends
-    <ul>
-      <li class="friend1">${userFriends[0]}</li>
-      <li class="friend2">${userFriends[1]}</li>
-      <li class="friend3">${userFriends[2]}</li>
-    </ul>
-  </li>
   <li class="user-info__step-goal">Daily Step Goal: ${currentUser.dailyStepGoal}</li>
-  <li class="user-info__average-step-goal">Average Daily Step Goal: ${userRepo.calculateAverageStepGoal()}</li>`
+  <li class="user-info__average-step-goal">Average Daily Step Goal: ${userRepo.calculateAverageStepGoal()}</li>
+  `;
+  userFriendsDisplay = printList(userFriendsDisplay, userFriends);
```

Comment:
Thanks for walking me through the way this printList() function is working!
This is great, and really dynamic - beautifully done 👏